Libraries/Image/Image.ios.js

```diff
@@ -118,6 +118,20 @@ var Image = React.createClass({
      * 'stretch': Scale width and height independently, This may change the
      * aspect ratio of the src.
      */
+    getSize: PropTypes.func,
+    /**
```

Comment:
I believe this is what generates the docs: https://github.com/facebook/react-native/tree/master/website/jsdocs

Agree we shouldn't put `getSize` into props because it's not a prop.

I don't think the generator can parse `statics` (yet)? I'd put the docs to `getSize` where they belong and hopefully we can improve the generator later, it's definitely not perfect. I often see people put all the API docs at the top, e.g. https://github.com/facebook/react-native/blob/master/Libraries/Utilities/Alert.js